### PR TITLE
Update grafana dashboard grouping time to match the default resolution of 5s in heapster

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

File: grafana/RELEASES.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Release Notes
 
+## 0.7 (Unreleased)
```
Comment (vmarmol) [inline grafana/RELEASES.md]: Was this meant to be here? Maybe do in another PR or commit? We should probably mention some of the GCM changes as well.

Comment (vishh) [inline grafana/RELEASES.md]: Note that it says unreleased. I am just tracking all the changes as we make
them under a new release label.

**Comment by vishh** [comment]
Self merging since tests do not capture grafana dashboards.
